Apps/WebClient/src/ClientApp/src/models/laboratoryOrderTimelineEntry.ts

```diff
@@ -118,10 +120,11 @@ export class LaboratoryTestViewModel {
         this.testName = model.batteryType;
 
         this.result = "Pending";
-        if (model.testStatus === statusFinal) {
+        if (model.testStatus === statusCompleted) {
             this.result = model.outOfRange ? resultOutOfRange : resultInRange;
+        } else if (model.testStatus === statusCancelled) {
+            this.result = statusCancelled;
         }
```

Comment:
@ninosamson Can you confirm that the complete set of battery statuses is `["Active", "Completed", "Cancelled"]`?

I'm trying to make sense of this criteria:

> - remove mapping for Status "Partial" = Result "Pending"

Reply:
It's in the description and in the ticket.  https://dev.azure.com/qslvic/Health%20Gateway/_workitems/edit/12821

Reply:
Correct - those are the possible statuses for the battery tests. We had bad test data before that provided a battery status of "Partial", but that shouldn't happen, so that criteria is more for cleanup. 

Also, can you guys confirm if we are doing anything with the order status?

Reply:
> Correct - those are the possible statuses for the battery tests. We had bad test data before that provided a battery status of "Partial", but that shouldn't happen, so that criteria is more for cleanup.
> 
> Also, can you guys confirm if we are doing anything with the order status?

Confirmed.